src/CLIPreChecker/DeprecationChecker/DeprecationChecker.ts

```diff
@@ -0,0 +1,67 @@
+import chalk from 'chalk'
+import { spawn } from 'child_process'
+import { join } from 'path'
+import { DeprecationCheckerStore, IDeprecationCheckerStore, VersionDeprecationInfo } from './DeprecationCheckerStore'
+
+export class DeprecationChecker {
+  private static readonly DEPRECATION_CHECK_INTERVAL = 1 * 3600 * 1000
```

Comment:
Did you intend to decrease the check interval besides fixing it?

Reply:
Yeah, I though it woudn't hurt doing it one time per hour